app/models/manageiq/providers/redhat/infra_manager/provision/configuration.rb

```diff
@@ -49,6 +51,10 @@ def configure_container
 
   private
 
+  def sysprep_specification_selected?
+    options.dig(:sysprep_enabled,1) == "Sysprep Specification"
```

Comment:
> sysprep_option = get_option(:sysprep_enabled)

Does `sysprep_option` contain `fields` or "Sysprep Specification"? Can we replace `dig` with it?